match/lib/match/encrypt.rb

```diff
@@ -93,7 +93,14 @@ def crypt(path: nil, password: nil, encrypt: true)
       command << "-out #{tmpfile.shellescape}"
       command << "-a"
       command << "-d" unless encrypt
-      command << "&> /dev/null" unless FastlaneCore::Globals.verbose? # to show an error message if something goes wrong
+      # to show an error message if something goes wrong
+      unless FastlaneCore::Globals.verbose?
+        if !Helper.windows?
+          command << "&> /dev/null"
+        else
+          command << "2> nul"
```

Comment:
Ouch, that is not working on Windows @lacostej :

Reply:
Tried all the obvious results from Google searches and Stackoverflow results - always "unknown option" in one form or another.